exporters/mongodb3/mongodb3.prometheus.json.tmpl

```diff
@@ -28,6 +28,9 @@
                     "version": "{{.integration_version}}",
                     "name": "{{.integration}}"
```

Comment:
I would add it as well in the template config the `scrape_timeout` commented out

Reply:
Updated here: [69223f2](https://github.com/newrelic/newrelic-prometheus-exporters-packages/pull/166/commits/69223f213f17bcc6fbc6cab3067cbae514633308)